clang/include/clang/Basic/Builtins.td

```diff
@@ -3636,6 +3648,22 @@ def Fmin : FPMathTemplate, LibBuiltin<"math.h"> {
   let OnlyBuiltinPrefixedAliasIsConstexpr = 1;
 }
 
+def FmaximumNum : FPMathTemplate, LibBuiltin<"math.h"> {
```

Comment:
Why is this a libbuiltin? IIUC this is only supposed to be an intrinsic, never alibrary function.

Reply:
Oh, newer libc does have this function: https://www.gnu.org/software/libc/manual/html_node/Misc-FP-Arithmetic.html

Reply:
In that case this should probably be a `GNULibBuiltin`. Or is it actually standardized?

Reply:
Yes. They are in C23: https://libc.llvm.org/c23.html